localstack/testing/snapshots/prototype.py

```diff
@@ -24,6 +24,10 @@
 SNAPSHOT_LOGGER.setLevel(logging.DEBUG if os.environ.get("DEBUG_SNAPSHOT") else logging.WARNING)
 
 
+def is_ultra_debug():
```

Comment:
is this already used somehwere?

Reply:
haha, no that was actually not intended to go into the PR :grin: 

But I was thinking of adding a way to basically  generate a raw format snapshot which might be useful when debugging transformer issues. It would then just save the raw snapshot (gitignored) along the transformed one